src/main/java/appeng/parts/reporting/PartConversionMonitor.java

```diff
@@ -72,7 +74,7 @@ public PartConversionMonitor( final ItemStack is )
 	}
 
 	@Override
-	public boolean onPartShiftActivate( final EntityPlayer player, final EnumHand hand, final Vec3d pos )
+	public boolean onPartActivate( EntityPlayer player, EnumHand hand, Vec3d pos )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') The Cyclomatic Complexity of this method "onPartActivate" is 11 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)